### DISCUSSION TITLE
Packaging product for easy management

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#640] [Ideas]
I am wondering if there is a plan to place all features of these wonder extensions into a single package that can be installed one time.  Or at the least package into a zip that will contain all the latest versions that can be used to install in VS CODE.

My reason for asking is it would be best if all of our developers (that want to use VS CODE) use the same tools.  This assists in learning the features and troubleshooting for those that know it well, not to mention giving the IT group in charge of updating software the ability to pass out any release updates easily through automated tools.

**Comment by worksofliam** [comment]
Great question! The only type of packing up we do is through the VS Code development pack: https://marketplace.visualstudio.com/items?itemName=HalcyonTechLtd.ibm-i-development-pack

**Reply by m-tyler** [reply]
That pack is just an informational download, which links to all the packages that should be installed.   I was wondering if all those packages could be packaged into that extension for a simpler download.  Or can the latest version be updated on that extension so that I can make quick comparisons?

**Reply by worksofliam** [reply]
@m-tyler as well as information, when you install the pack it will install all the latest versions of those extensions.

We will never be able to package everything up, other than the use through the extension pack.

**Reply by m-tyler** [reply]
I have seen that but I just hoping that there was an easier way to get updates and have them installed.  Mycompany does not (normally) allow for software and add-ons to be installed by the users and needs to go through the IT group responsible for managing software on users' computers (I and several others have an exception to this rule).  

My company blocks marketplace access so users cannot install their own add-ons without asking.   This means that things have to be installed using the Software Center app to push new updates to users.   We tried with RDi and the add-ons we use but that is difficult to get right  but with VS Code it looks like we could script a simple batch script for updates.   But this would be simpler to get going if there was just a single down load (not necessarily a single install item).

Anyway, I guess I will look to see if I can script an auto-download of these add-ons.   then have it attempt to get the add-ons every week.
